src/main/java/com/google/cloud/tools/maven/CloudSdkLoginMojo.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2018 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.maven;
+
+import com.google.cloud.tools.appengine.api.AppEngineException;
+import com.google.cloud.tools.appengine.cloudsdk.CloudSdk;
+import com.google.cloud.tools.appengine.cloudsdk.CloudSdkAuth;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Mojo;
+
+@Mojo(name = "cloudSdkLogin")
+public class CloudSdkLoginMojo extends CloudSdkMojo {
+
+  @Override
+  public void execute() throws MojoExecutionException, MojoFailureException {
+    try {
+      CloudSdk sdk = getAppEngineFactory().defaultCloudSdkBuilder().build();
+      new CloudSdkAuth(sdk).login();
```

Comment:
can you add in that warning if serviceAccountKeyFile is specified? https://github.com/GoogleCloudPlatform/app-gradle-plugin/pull/236/files#diff-61ff2d5be7163076a4c1810bd162f73eR166